Makefile

```
@@ -18,6 +18,10 @@ testacc:
 	@echo "==> Running acceptance tests..."
 	TF_ACC=1 AAP_HOST="https://localhost:8043" AAP_INSECURE_SKIP_VERIFY=true go test -count=1 -v ./...
 
+testacc-aapdev:
+	@echo "==> Running acceptance tests..."
+	TF_ACC=1 AAP_HOST="http://localhost:9080" AAP_INSECURE_SKIP_VERIFY=true go test -count=1 -v ./...
```

Nitpick but if we're connecting over `http` and not `https`, the `AAP_INSECURE_SKIP_VERIFY` flag isn't necessary. See https://github.com/ansible/terraform-provider-aap/pull/61#issuecomment-2854833869

```suggestion
	TF_ACC=1 AAP_HOST="http://localhost:9080" go test -count=1 -v ./...
```